WORKSPACE

```diff
@@ -40,6 +40,10 @@ load("//proto:repositories.bzl", "rust_proto_repositories")
 
 rust_proto_repositories()
 
+load("//bindgen:repositories.bzl", "rust_bindgen_repositories")
+
+rust_bindgen_repositories()
```

Comment:
@damienmg  already mentioned this concern in his high-level comment, but I'll reiterate it here:

---

Whats the suggestion for users who want fine grained control of their cargo deps?

For reference, our suggestion for the similar proto/grpc case is [available here](https://github.com/bazelbuild/rules_rust/tree/master/proto#customizing-dependencies).

Reply:
Documentation for this toolchain is basically identical, I will paste it and make the relevant edits.